### PR TITLE
Update nl.yml

### PR DESCRIPTION
**Pull request by Xelium**
Updated Dutch Translations

### CODE REVIEW

**Comment by Xelium** [comment]
I know shit about GitHub, this is the first time I try to participate to a project. So please check if what I am doing is correct. I don't want to fuck something up.

File: Minepacks/resources/lang/nl.yml [hunk 1]
```diff
@@ -1,28 +1,122 @@
+Version: 2.0
```
Comment (GeorgH93) [inline Minepacks/resources/lang/nl.yml:1]: The language file version should not be changed by hand.
It is used by the plugin to figure out weather or not it has to update the language file.

**Comment by GeorgH93** [comment]
Thanks for updating the nl language file.
Please change the version back to 18 than I will merge it.

**Comment by Xelium** [comment]
> Thanks for updating the nl language file.
> Please change the version back to 18 than I will merge it.

I think I just did. Can you check?

**Comment by GeorgH93** [comment]
Yes, you did. Thank you very much.

**Pull request merged**